### PR TITLE
NO-ISSUE add skipper mac file

### PR DESCRIPTION
**Pull request by michaellevy101**

### CODE REVIEW

**Comment by gamli75** [comment]
@michaellevy101 can you please explain why you added it?

**Review by nmagnezi — COMMENTED** [review]
I would add something about this in `README.md` for visibility, and also a one-liner comment to this file - so only people who run skipper using macOS will know to use this file (also worth mentioning the minimal skipper version this will work with).

Thanks for fixing skipper for macOS users!

**Review by nmagnezi — COMMENTED** [review]
I also see a few lines missing, such as `CRD_OPTIONS` etc


See https://www.diffchecker.com/iZ5De8pi

**Comment by michaellevy101** [comment]
@gamli75 The default skipper file doesn't work on MAC (The docker implementation is Docker desktop and not the libvirt) and skipper does not support changing the volume mounts.

**Comment by michaellevy101** [comment]
@nmagnezi I updated the file and added all the new flags

**Comment by gamli75** [comment]
@michaellevy101 who will update this file if someone will add a new env parameter?

File: skipper-mac.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+build-container-image: assisted-service-build
+
+containers:
+    assisted-service-build: Dockerfile.assisted-service-build
+volumes:
+    - $HOME/.cache/go-build:/go/pkg/mod
+    - $HOME/.cache/golangci-lint:$HOME/.cache/golangci-lint
+    - $HOME/.minikube:$HOME/.minikube
+    - $HOME/.docker/:$HOME/.docker
+env:
```
Comment (eranco74) [inline skipper-mac.yaml:11]: I'm guessing this is a duplicate of the skipper.yaml
Consider extracting it to a separate file and use the env-file option
https://github.com/Stratoscale/skipper#environment-variables

**Pull request merged**